Expensify/App
# fix: remove extra space between paragraph text

### Details

Remove extra space between the two paragraphs in concierge messages.
### Fixed Issues

$ https://github.com/Expensify/App/issues/7474

### Tests

1. Open a chat with Concierge.
2. Type "I can't delete workspace". The replied messages contain multiple paragraphs.
3. There should be a normal spacing between the paragraphs.

### QA Steps

1. Same as test steps

### Tested On


### Screenshots


#### Web

<img width="1440" alt="webb" src="https://user-images.githubusercontent.com/77761491/152695462-ade7b21f-55cd-4e26-8a64-fd937468adc0.png">

#### Mobile Web

![mwebb](https://user-images.githubusercontent.com/77761491/152695485-6ef3c227-0025-45c8-98b4-5c584cc16c97.png)

#### Desktop

<img width="1440" alt="desktopp" src="https://user-images.githubusercontent.com/77761491/152695507-528d38c9-e35a-4433-ae19-081902f5905a.png">

#### iOS

![ioss](https://user-images.githubusercontent.com/77761491/152695526-e32c3f78-4ec2-4f43-8f9a-f421dc96595f.png)

#### Android

![Screenshot_1644774448](https://user-images.githubusercontent.com/77761491/153767709-e8b5d854-38da-4ec4-bdfc-74bcbed9813c.png)
